kickstart/provision.erb

```diff
@@ -151,10 +152,11 @@ salt-call --no-color --grains >/dev/null
 
 sync
 
+<% if host_provision -%>
 # Inform the build system that we are done.
 echo "Informing Foreman that we are built"
 wget -q -O /dev/null --no-check-certificate <%= foreman_url %>
-# Sleeping an hour for debug
```

Comment:
Removing this comment sounds like a good change in itself since it's a leftover debug comment.